README.md

```diff
@@ -70,6 +70,13 @@ production:
   password: <%= ENV['circuitverse_DATABASE_PASSWORD'] %>
 ```
 
+- If you are facing difficulties installing RVM which is primarly     because Ubuntu's desktop edition ships with a version of rvm which is not the exact rvm version hence before installing it(RVM) you need to get rid of this inbuilt version  
```

Comment:
What is meant by "exact version" of rvm? 
Also is it possible to break this line into smaller ones?

Reply:
By that I meant to imply that it is an older version

Reply:
I will try to shorten the line thanks for the feedback